lib/grandpa/vote_message_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build integration
```

Comment:
shouldn't this be renamed to `vote_message_integration_test.go`?

Reply:
Indeed! Done.